sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SetupAccountActivity.kt

```diff
@@ -47,13 +51,31 @@ class SetupAccountActivity : DuckDuckGoActivity(), SetupFlowListener {
     private val binding: ActivitySyncSetupAccountBinding by viewBinding()
     private val viewModel: SetupAccountViewModel by bindViewModel()
 
+    private lateinit var screen: Screen
```

Comment:
shouldn't this be in the ViewModel? any reason you are not using https://developer.android.com/topic/libraries/architecture/viewmodel/viewmodel-savedstate instead?

Reply:
Since it's a param the activity receives from the caller, and we already have logic to extract it from the bundle, I prefer to save it as well on the `Activity`. It also allows me to pass it to the ViewModel in the same way as I do when is part of the original intent.